exercises/series/src/example.c

```diff
@@ -0,0 +1,24 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "series.h"
+
+SeriesResults_t *series(char *inputText, unsigned int substringLength)
+{
+   int substringCount = strlen(inputText) - (substringLength - 1);
+   SeriesResults_t *results = malloc(sizeof(SeriesResults_t));
```

Comment:
I don't think it's really necessary to `malloc` this structure. It is fixed size and so can be returned by value.